src/main/java/com/lambdaschool/oktafoundation/SeedData.java

```diff
@@ -204,22 +204,35 @@ public void run(String[] args) throws
         c6.getActivities().add(new ClubActivities(c6, a5));
         c6.getActivities().add(new ClubActivities(c6, a7));
         c6.getActivities().add(new ClubActivities(c6, a8));
-        clubService.save(c6);
+        c6 = clubService.save(c6);
+
```

Comment:
A lot of work was put into this and it shows that you give a damn.  It is very organized and well laid out,